content/grants/dusko-valve.md

```diff
@@ -0,0 +1,81 @@
+
+2. The second phase of Valve will be replacing the planetine backend with Tirrel’s planet distribution tool. At this point Valve will now be open source and I will work with Tlon / Tirrel to create a guide that explains how NFT creators customize Valve to distribute planets for their specific contract.
+
+## Responsibilities by Organization
+
+### Tlon
+
+Tlon will help with managing the integration of planettine for the backend of Valve’s MVP. By managing, I mean working with me to polish onboarding copy for the frontend. In addition, they will host the planets of the Dusko community and help with onboarding — providing learning sessions to teach people how to onboard and essentially assist in general hand-holding of the community if necessary. They will also provide marketing help to promote the Dusko project, as well as additional onboarding and promotional support to the other artists who will use Valve initially in partnership with this grant.
+
+### Tirrel
+
+Tirrel will work with me to develop the final version of Valve as an open source tool kit for NFT creators to customize and use to distribute planets among those holding their NFTs. Their backend will replace the planettine backend in phase 2.
+
+### Urbit Foundation
+
+The foundation will provide 500 planets to be used and distributed by artists to onboard their communities into Urbit. The first 181 will be for Dusko’s community, while the additional will be gifted to additional artists who I work with who want to onboard their communities to Urbit and use groups as their community’s social hub.
+
+## User Story
+
+I’m grateful I have access to a platform which simplifies the experience of interacting with my community.
+
+A tool for me, as an artist selling artwork, to provide my community immediate access to an aesthetic and easy to use platform (Urbit) allows me to legitimize the use of NFTs and use them myself.
+
+## Milestones
+
+### Milestone 1 - MVP
+
+Payment: 1 star
+
+Completion: 30 Apr 2022
+
+MVP: a web page gated using Metamask Protocol which users are granted access to once they mint a corresponding NFT.
+
+### Milestone 2 - Airtable form and Tirrel Backend
+
+Payment: 1 star
+
+Completion: 30 Jun 2022
+
+Airtable form for Valve which other artists fill out to request redeemable planets and at least two artists lined up to transition their community over to Urbit through a Valve NFT release.
+
+Implementation of Tirrel backend and open-sourcing of Valve through release of repo and guide.
+500 planets distributed in total with at least communities transitioned over to Urbit.
```

Comment:
"...at least communities..."

I feel like something is missing there.

Reply:
Looks like it should be 'at least **_2_** communities'? Good catch, thanks.